input/2017-bae-europe/recipes.xml

```diff
@@ -0,0 +1,941 @@
+
```

Comment:
having these in their own file is an excellent choice. Would you mind adding a comment at the beginning of each recipe indicating the reference where it can be found?